sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager_base.py

```diff
@@ -131,11 +122,3 @@ def _calculate_backoff(self, attempts: int) -> float:
         return min_backoff_milliseconds + (
             random.uniform(0.0, 1.0) * (calculated_milliseconds - min_backoff_milliseconds)  # nosec
         )
-
-    def __eq__(self, other):
```

Comment:
Do we not need this equality overload?

Reply:
It was added at some point, but we never use it.